Content.Server/Cargo/Systems/PriceGunSystem.cs

```diff
@@ -25,10 +27,15 @@ protected override bool GetPriceOrBounty(EntityUid priceGunUid, EntityUid target
         else // Otherwise appraise the price
         {
             var price = _pricingSystem.GetPrice(target);
-            _popupSystem.PopupEntity(Loc.GetString("price-gun-pricing-result", ("object", Identity.Entity(target, EntityManager)), ("price", $"{price:F2}")), user, user);
+            _popupSystem.PopupEntity(Loc.GetString("price-gun-pricing-result",
+                    ("object", Identity.Entity(target, EntityManager)),
+                    ("price", $"{price:F2}")),
+                user,
+                user);
         }
 
-        _useDelay.TryResetDelay((priceGunUid, useDelay));
+        _audio.PlayPvs(entity.Comp.AppraisalSound, target);
```

Comment:
```suggestion
        _audio.PlayPvs(entity.Comp.AppraisalSound, entity.Owner);
```
Did some testing, and found a minor nitpick. The sound should originate from the apraisal tool's location not from the target, otherwise the loudness of the sound will vary depending on the distance.
I'll make a quick commit for you.